### PR TITLE
Hometask 1

### PR DESCRIPTION
**Pull request by sergbondarenko**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* Hometask1 */
+
 let usersList = [];
 
 const MyTable = ({ MyName, MySurname, MyLastname}) => {
```
Comment (pannkotsky) [inline app.js]: use `camelCase` for object keys please (first letter is not capitalized)

File: app.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* Hometask1 */
+
 let usersList = [];
```
Comment (pannkotsky) [inline app.js]: global variables shouldn't be used for data, pass it as a property to component where it should be used

**Review by pannkotsky — CHANGES_REQUESTED** [review]
I don't think your task is ready. You don't render the table of users.
usersList (or one user for now) should be stored in state of App component, changed in Form's submit and passed as a property to Table. Check https://github.com/amtoss-study/react-study-ssuprun or https://github.com/amtoss-study/react-study-askrinnik for reference, they got it right

File: app.js (hunk 2 above)
Comment (sergbondarenko) [reply app.js]: Done

File: app.js (hunk 1 above)
Comment (sergbondarenko) [reply app.js]: Done

**Pull request merged**